db/migrate/20160203145841_add_content_id_to_about_page.rb

```diff
@@ -0,0 +1,16 @@
+require "securerandom"
+
+class AddContentIdToAboutPage < ActiveRecord::Migration
+  def up
+    add_column :about_pages, :content_id, :string
+
+    AboutPage.reset_column_information
+    AboutPage.all.each do |about_page|
+      about_page.update_attribute(:content_id, SecureRandom.uuid)
```

Comment:
Data migrations are definitely a good choice where there's interaction with another system. So republishing to publishing-api or panopticon, for example. This is because developers and CI need to be able to run normal migrations without requiring other services to be running.

I'm comfortable with this change being done in a conventional migration. It means it will happen by default, rather than requiring the person deploying to (remember to) manually run a rake task.

@benlovell when you say:

> atomically in the migration

If I understand correctly, migrations against MySQL aren't atomic, but maybe I've got the wrong end of the stick?